src/screens/ChannelListScreen/index.tsx

```diff
@@ -53,12 +71,26 @@ const ChannelListScreen = () => {
   };
 
   return (
-    <FlatList
-      data={channels}
-      keyExtractor={(item) => item.id}
-      scrollEnabled={false}
-      renderItem={renderChannelItem}
-    />
+    <>
+      <StatusBar translucent={false} />
+      <View style={{height: 52}}>
+        <Search route={route} onPress={goToContactScreen} />
+      </View>
+
+      <FlatList
+        data={channels}
+        keyExtractor={(item) => item.id}
+        ListHeaderComponent={
+          <ChannelListHeaderItem
+            name={`@${profile?.username}`}
+            picture={profile?.profile_pic_path}
+            type="SIGNED"
+            testID="horizontal-tab-0"
+          />
+        }
+        renderItem={renderChannelItem}
+      />
+    </>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `StatusBar` and a `Search` component above the `FlatList`. The `Search` component receives a `route` prop and an `onPress` prop which is set to `goToContactScreen`. Ensure that the `route` prop is being passed correctly and that the `goToContactScreen` function behaves as expected when passed to the `Search` component.

The `FlatList` now includes a `ListHeaderComponent`, which is set to a `ChannelListHeaderItem`. Ensure that the `profile` object has the expected shape and contains the `username` and `profile_pic_path` properties. If there's a chance that these might not be defined, consider providing default values.

```diff
    <>
      <StatusBar translucent={false} />
      <View style={{height: 52}}>
        <Search route={route} onPress={goToContactScreen} />
      </View>

      <FlatList
        data={channels}
        keyExtractor={(item) => item.id}
        ListHeaderComponent={
          <ChannelListHeaderItem
            name={`@${profile?.username}`}
            picture={profile?.profile_pic_path}
            type="SIGNED"
            testID="horizontal-tab-0"
          />
        }
        renderItem={renderChannelItem}
      />
    </>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->